include/oneapi/mkl/blas/detail/cublas/blas_ct.hpp

```diff
@@ -16,717 +16,673 @@
 *  limitations under the License.
 *
 **************************************************************************/
+
 //
-// Generated based on onemkl/blas/blas.hpp
+// Generated based on include/oneapi/mkl/blas/detail/blas_ct_templates.hpp
 //
 
-#ifndef _DETAIL_CUBLAS_BLAS_HPP_
```

Comment:
I believe in other header files we are using macro guards, does it make sense to keep it aligned with them?